tests/test1/main.tf

```
@@ -43,7 +43,7 @@ resource "random_string" "sqs_rstring" {
 }
 
 module "vpc" {
-  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork?ref=v0.0.1"
+  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork"
```

To save us from having to update later can we explicitly pin to master.  Reasoning is when style linting with tuvok is implemented, this will fail the module pinning test.

```suggestion
  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork?ref=master"
```

---

This one should be fixed along with another line